Makefile

```diff
@@ -91,9 +91,11 @@ test-vectors: pull-serialization-tests run-vectors
 
 # Test all without the submodule test vectors with release configuration
 test:
-	cargo nextest run --all --exclude serialization_tests --exclude forest_message --exclude forest_crypto
+	cargo nextest run --all --exclude serialization_tests --exclude forest_message --exclude forest_crypto --exclude forest_db
 	cargo nextest run -p forest_crypto --features blst --no-default-features
 	cargo nextest run -p forest_message --features blst --no-default-features
+	cargo nextest run -p forest_db --no-default-features --features paritydb
```

Comment:
This is another reason why I don't like mutually exclusive features, it bloats testing scripts

Reply:
Ya, on top of it it's not really the first one, we already have `forest_fil_cns` and `forest_deleg_cns`. They just don't have any test coverage. :clown_face: